ui/public/locales/en.json

```diff
@@ -640,8 +640,6 @@
 "label.create.account": "Create Account",
 "label.create.backup": "Start Backup",
 "label.create.network": "Create New Network",
-"label.create.network.gateway.description": "The gateway of the tier in the super CIDR range and not overlapping the CIDR of any other tier in this VPC.",
-"label.create.network.netmask.description": "Netmask of the tier. For example, with VPC CIDR of 10.0.0.0/16 and network tier CIDR of 10.1.1.0/24, gateway is 10.1.1.1 and netmask is 255.255.255.0",
```

Comment:
@davidjumani I found the keys, labels are not translated in Networks -> VpcTiersTab. It doesn't seem right that you delete these keys?
![image](https://user-images.githubusercontent.com/13766648/129692553-60ec07e3-ab62-4b8e-8a45-fd7b6e20f622.png)

Reply:
@utchoang Forgot to add the vue file. Can you please check now